pandas/io/pytables.py

```
@@ -303,6 +303,9 @@ def read_hdf(path_or_buf, key=None, **kwargs):
 
         """
 
+    if kwargs.get('mode') == 'w':
```

Should raise other invalid mode here?